src/test/java/org/gridsuite/study/server/VoltageInitTest.java

```diff
@@ -32,6 +32,7 @@
 import org.gridsuite.study.server.repository.networkmodificationtree.NetworkModificationNodeInfoRepository;
 import org.gridsuite.study.server.repository.sensianalysis.SensitivityAnalysisParametersEntity;
 import org.gridsuite.study.server.service.*;
+import org.gridsuite.study.server.service.securityanalysis.SecurityAnalysisService;
```

Comment:
Just one remark : 
This import should be useless in this test file, because we only test voltage init here ...
Same for the SensitivityAnalysisService and ShortCircuitService, who we don't need to autowire ...

Reply:
I agree, we don't even use those services
Not doing it here though if it's ok